lib/kapost_deploy/plugins/notify_honeybadger_after_promote.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "kapost_deploy/heroku/app_releases"
+
+module KapostDeploy
+  module Plugins
+    # After-promotion plugin to notify honeybadger after a promotion is complete.
+    # Honeybadger.io can be configured to clear all honeybadgers upon this deploy notification.
+    class NotifyHoneyBadgerAfterPromote
```

Comment:
Curious about the `AfterPromote` suffix, the use of this plugin could potentially be used for before and after hooks? If so maybe the suffix isn't necessary.

You might want to use `Honeybadger` without the capital `B` since they don't use it in their documentation.